### DISCUSSION TITLE
Hello New Here ✋🏻

### DISCUSSION THREAD

**Discussion by WayneAyoAce** [community/community#147567] [New to GitHub] [answered]
Well just started using the platform,
I have been able to figure out a thing or two by myself and I think I got to make out something lol
Not sure
But I'd love and appreciate some help and quality improvement if possible
Thanks💖

**Comment by tuves** [comment]
Hi @WayneAyoAce, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by devfemibadmus** [comment, marked as answer]
Hi @WayneAyoAce you're welcome

![image](https://github.com/user-attachments/assets/3d4e72a2-8333-4f10-beef-14849bb600db)

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @WayneAyoAce 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Reply by Synclair27** [reply]
I have Aphasia.
